trace/cloud-trace-demo-app-opentelemetry/README.md

```diff
@@ -0,0 +1,73 @@
+# cloud-trace-demo-app-opentelemetry
+
+Open this demo app in [Google Cloud Shell](https://cloud.google.com/shell/docs/). This includes necessary tools.
+
+
+[![Open Cloud Trace Demo APP in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=trace/cloud-trace-demo-app/README.md&amp;cloudshell_tutorial=trace/cloud-trace-demo-app/README.md)
+
+#### Demo Requirements
+If you are using Cloud Shell, skip to the next section.
+
+1. Install gcloud <https://cloud.google.com/sdk/install>
+2. Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>
+3. Install docker <https://docs.docker.com/install/>
+
+    
+#### Create a GKE cluster
+
+8. Enable Google Cloud and set up region and zone.
+
+    `gcloud init`
+9. Enable the GKE API & billing:
+
+    `gcloud services enable container.googleapis.com`
+10. Create a GKE cluster named "cloud-trace-demo", replace you GCP zone with
+your gcp-zone closest in proximity to you:
```

Comment:
Is there a page that lists the available gcp-zones? If we can link to it here then that would help the reader complete this step faster.